runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/GameTimer.java

```diff
@@ -52,6 +52,14 @@ String getTime(boolean waveTime)
 		return formatTime(LocalTime.ofSecondOfDay(elapsed.getSeconds()));
 	}
 
+	double getPBTime()
```

Comment:
you could probably just return a `Duration` here too

Reply:
I preferred returning just the `double` value as its what is getting saved to and retrieved from the configuration filed